modules/tiles/src/tileset/helpers/bounding-volume.js

```diff
@@ -2,24 +2,37 @@
 // See LICENSE.md and https://github.com/AnalyticalGraphicsInc/cesium/blob/master/LICENSE.md
 
 /* eslint-disable */
-import {Quaternion, Vector3, Matrix3, Matrix4, degrees} from '@math.gl/core';
-import {BoundingSphere, OrientedBoundingBox} from '@math.gl/culling';
-import {Ellipsoid} from '@math.gl/geospatial';
+import {Vector3, Matrix3, Matrix4} from '@math.gl/core';
 import {assert} from '@loaders.gl/loader-utils';
+import * as mat4 from 'gl-matrix/mat4';
 
-// const scratchProjectedBoundingSphere = new BoundingSphere();
+const scratchVector = new Vector3();
+const transformScratchVectorU = new Vector3();
+const transformScratchVectorV = new Vector3();
+const transformScratchVectorW = new Vector3();
 
-function defined(x) {
-  return x !== undefined && x !== null;
+function BS_transform(obj, transform) {
```

Comment:
We don't use underscores, or abbreviations (unless very common). We use `verbNoun` convention for functions. `transformBoundingVolume`?

Reply:
Please add JSDoc headers to all functions.

Reply:
ok, I'll fix it. This is temporary functions. The next steep will be PR for math.gl (it has done)

Reply:
BS_transform --> transformBoundingSphere
OBB_transform --> transformOrientedBoundingBox
ok?

Reply:
Love it!